src/KafkaFlow.OpenTelemetry/ActivitySourceAccessor.cs

```diff
@@ -19,7 +16,9 @@ internal static class ActivitySourceAccessor
 
     internal static void SetGenericTags(Activity activity, IEnumerable<string> bootstrapServers)
     {
-        activity?.SetTag(Conventions.AttributeMessagingSystem, MessagingSystemId);
-        activity?.SetTag(Conventions.AttributePeerService, string.Join(",", bootstrapServers ?? Enumerable.Empty<string>()));
+        // https://opentelemetry.io/docs/languages/net/libraries/#note-on-versioning
+        // https://github.com/open-telemetry/opentelemetry-dotnet/blob/core-1.9.0/src/Shared/SemanticConventions.cs
+        activity?.SetTag("message.type", MessagingSystemId);
```

Comment:
According to the guildines `messaging.type` does not exist as an attribute and `messaging.system` should be kept.
https://opentelemetry.io/docs/specs/semconv/messaging/messaging-spans/